How-To-Setup-Internal-Package-Repository.md

```diff
@@ -271,35 +256,34 @@ Jenkins requires several PowerShell scripts to automate the processes. Create a
   . .\ConvertTo-ChocoObject.ps1
 
   # get all of the packages from the test repo
-  $testPkgs = choco list --source $TestRepo | Select-Object -Skip 1 | Select-Object -SkipLast 1 | ConvertTo-ChocoObject
-  $prodPkgs = choco list --source $ProdRepo | Select-Object -Skip 1 | Select-Object -SkipLast 1 | ConvertTo-ChocoObject
+  $testPkgs = choco list --source $TestRepo -r | ConvertTo-ChocoObject
+  $prodPkgs = choco list --source $ProdRepo -r | ConvertTo-ChocoObject
   $tempPath = Join-Path -Path $env:TEMP -ChildPath ([GUID]::NewGuid()).GUID
+
   if ($null -eq $testPkgs) {
       Write-Verbose "Test repository appears to be empty. Nothing to push to production."
   }
+
   elseif ($null -eq $prodPkgs) {
       $pkgs = $testPkgs
   }
+
   else {
       $pkgs = Compare-Object -ReferenceObject $testpkgs -DifferenceObject $prodpkgs -Property name, version | Where-object SideIndicator -eq '<='
   }
 
   $pkgs | ForEach-Object {
       Write-Verbose "Downloading package '$($_.name)' to '$tempPath'."
-      choco download $_.name --no-progress --output-directory=$tempPath --source=$TestRepo
-
+      choco download $_.name --no-progress --output-directory=$tempPath --source=$TestRepo --ignore-dependencies
       if ($LASTEXITCODE -eq 0) {
           $pkgPath = (Get-Item -Path (Join-Path -Path $tempPath -ChildPath '*.nupkg')).FullName
-

```

Comment:
Can you add this blank lines back in - they help make it visually obvious for the next comments to insert the test code.

Reply:
Addressed